Correct a spelling error

jacobherrington opened the pull request:
### Summary

My spellchecker flagged this as an incorrect spelling, upon further
research it appears to be a point of contention in English. Either way
might work.

gmcgibbon reviewed (commented):
I prefer `queueing` but it seems we have more results for `queuing` (including file/class names), so I think this is the right approach. Can you please find replace the rest of the places in our docs where we use `queueing` instead of `queuing`?

Also, please use `[ci skip]` for documentation PRs. Thanks! 😄

jacobherrington commented:
@gmcgibbon I made the change, but stuck to markdown files. There seems to be a few places that the `queueing` spelling is used both in comments and in a couple of errors in `activejob`. 

Changing the comments doesn't seem like it would cause any problems, but the errors make me a bit nervous as some people might be referencing those in their own code.

How do you feel about me making those changes in comments and in the error messages?

Specifically:

https://github.com/rails/rails/blob/b2eb1d1c55a59fee1e6c4cba7030d8ceb524267c/activejob/lib/active_job/queue_adapters/sneakers_adapter.rb#L33

https://github.com/rails/rails/blob/48c287d1963fba09cf620527d35519f67d5ca01a/activejob/lib/active_job/queue_adapters/inline_adapter.rb#L19

gmcgibbon commented:
@jacobherrington You can change the comments without CI runs, but not error strings. Regardless, please squash your commits and I can merge what you have. Or, if you want to change the comments, I'll accept that too.

jacobherrington commented:
@gmcgibbon Changes made. I'll make a separate PR for the error strings and discuss the merits of that change there (if there are any).

gmcgibbon commented on actionview/test/ujs/public/vendor/jquery-2.2.0.js:
```diff
@@ -6852,7 +6852,7 @@ jQuery.speed = function( speed, easing, fn ) {
 	}
 
-	// Queueing
+	// Queuing
```
This is a vendor file we didn't create, I don't think we should change this.

jacobherrington replied:
My mistake, good catch!

gmcgibbon reviewed (commented):
One last comment, otherwise looks good 👍

jacobherrington committed:
Change queueing to queuing in docs and comments [skip ci]

My spellchecker flagged this as an incorrect spelling, upon further
research it appears to be a point of contention in English. Either way
might work.

After further examination queuing is much more common in the Rails
codebase so making this change will serve to standardize the spelling.

gmcgibbon commented:
You missed one ref in `railties/RDOC_MAIN.rdoc`. You can include that in your next patch.
